producers/smd_producer.py

```diff
@@ -252,8 +253,9 @@ def global_except_hook(exctype, value, exc_traceback):
 parser.add_argument('--default', help='store only minimal data', action='store_true', default=False)
 parser.add_argument('--image', help='save everything as image (use with care)', action='store_true', default=False)
 parser.add_argument('--tiff', help='save all images also as single tiff (use with even more care)', action='store_true', default=False)
-parser.add_argument("--postRuntable", help="postTrigger for seconday jobs", action='store_true', default=True)
+parser.add_argument("--postRuntable", help="postTrigger for seconday jobs", action='store_true', default=False)
```

Comment:
Isn't that what is used for the summary plots? Why default to False?

Reply:
Yes, that is why the default jobs set this flag. Adding a flag cannot set something to false, so this way is better: reprocessing/testing now won't overwrite the entries in the elog for the processing time etc.